backend/app.py

```
@@ -177,6 +177,24 @@ async def get_user_items(username: str):
+    try:
+        # Delete from SQL database
+        deleted = db_delete_item(item_id, username)
+        
+        if deleted:
+            # Delete from Qdrant vector database
+            delete_item_vector(item_id)
+            return {"success": True, "message": "Item deleted successfully"}
+        else:
+            return {"success": False, "error": "Item not found or unauthorized"}
+            
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)